web/themes/new_weather_theme/templates/block/block--weathergov-current-conditions.html.twig

```diff
@@ -28,9 +28,11 @@
       </div>
 
       <div class="display-flex margin-bottom-2">
+        {% if content.icon.icon %}
         <div class="icon margin-right-105 maxw-5 maxh-5">
-          {{ source(directory ~ "/assets/images/weather/icons/conditions/" ~ content.icon )}}
+          {{ source(directory ~ "/assets/images/weather/icons/conditions/" ~ content.icon.icon )}}
         </div>
+        {% endif %}
```

Comment:
We should switch to the link:href svg from the spritesheet here right?

Reply:
I think so, yeah.